src/cache.ts

```diff
@@ -0,0 +1,24 @@
+import * as request from 'request';
+import { reportError } from './shared/SentryConnector';
+const APP_STATUS = process.env.APP_STATUS_URI;
+export class ServerlessCache {
+     private static _cacheValue: object;
+
+     static set cachedValue(val) {
+        this._cacheValue = val;
+     }
+     static get cachedValue() {
+         return this._cacheValue;
+     }
+
+    static fillCache() {
+        try {
+            request(APP_STATUS, function(error, response, body) {
```

Comment:
I think that the request API might be stream based, so the try catch won’t actually wrap the error. We should try attaching an event listener for errors.

Reply:
Resolved with 80dde11